.env.example

```diff
@@ -11,6 +11,7 @@ NEXTAUTH_URL=http://localhost:3000
 ## Create a GitHub OAuth app here: https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app
 AUTH_GITHUB_ID=XXXXXXXX
 AUTH_GITHUB_SECRET=XXXXXXXX
+# you will be asked for a github callback url. it should be https://YOUR_DOMAIN.com/api/auth/callback/github
```

Comment:
Let's point to the official documentation instead https://authjs.dev/reference/core/providers_github#callback-url

Reply:
```suggestion
# you will be asked for a github callback url. it should be https://YOUR_DOMAIN.com/api/auth/callback/github. See docs: [authjs.dev/reference/core/providers_github#callback-url](https://authjs.dev/reference/core/providers_github#callback-url)
```

Reply:
Could you apply this change? I don't have permission to do so. :pray: 

```suggestion
# When setting up the OAuth app, you will be asked for a GitHub callback URL. It should be https://YOUR_DOMAIN.com/api/auth/callback/github. See docs: https://authjs.dev/reference/core/providers_github#callback-url
```